tests/kuttl/tests/octavia_scale/04-assert.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# Check for:
+#
+# - 0 OctaviaAPI CR
```

Comment:
Should this be `1 OctaviaAPI CR` since we're only scaling-down replicas and not deleting the CR?

Reply:
Yes, that should be `1OctaviaAPI CR with 0 replicas`. Thanks, fixed.